script/upgrades/dependencies.json

```diff
@@ -1,10 +1,10 @@
 {
   "42220": {},
   "62320": {
-    "USDCet": "0xfD46b16f86449b3e03e516A3632e1919A5CA1A53",
```

Comment:
@bowd is this useful for anything or was it only a mock token that was deployed at some point? we couldn't find any use for it so we removed it

Reply:
not sure if it's useful but the baklava deployment script fails if this is deleted, can you put it back in?

Reply:
How did you ran it? It doesn't fail for us without it

Reply:
yup the script is loading `USDCet` so now it works fine